.travis.yml

```diff
@@ -1,28 +1,35 @@
 language: php
 
 php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
-    - hhvm
+  - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
+  - hhvm
```

Comment:
well, they do because 2 spaces is the default indentation for YAML in the ruby library